clown/common/package.json

```diff
@@ -4,7 +4,8 @@
     "pre-push": "# OVERRIDE",
     "pre-commit": "yarn common-check",
     "lint-staged": "lint-staged",
-    "common-check": "run-p -c --aggregate-output validate-filenames 'clown check' check-integrity",
+    "check-prettier": "prettier './**/*.{t,j}s{,x}' '**/*.{css,scss,json,md}' --list-different",
```

Comment:
I remember you said that you didn't wanna do this because it's slow. Is that no longer the case?

Reply:
I don't now what changed exactly, but it's super fast now. Could be the newer version of the fact that I was not ignoring `node_modules` and `.git` 😂